config/manager/manager.yaml

```diff
@@ -25,9 +25,9 @@ spec:
         name: manager
         resources:
           limits:
-            cpu: 100m
-            memory: 30Mi
+            cpu: 500m
```

Comment:
should we give it a bit more CPU for spikes in the load? like 1000m to give it full CPU?

Reply:
cool with me, but I may make that just a limit. don't want to overallocate resources if we don't have to.

Reply:
fixed in amended commit.